docker/build.sh

```diff
@@ -146,6 +170,8 @@ manifest() {
   echo "Manifest list:"
   docker manifest inspect "${IMAGE_TAG}"
   docker manifest push --purge "${IMAGE_TAG}"
+  docker manifest inspect "${CRD_IMAGE_NAME}"
```

Comment:
All these should be `CRD_IMAGE_TAG`? Did you get a chance to validate this locally?

Reply:
Done. Yup I tested it locally by push it to my dockerhub registry